content/doc/book/blueocean/getting-started.adoc

```diff
@@ -79,8 +80,23 @@ If this is a new Jenkins instance, Blue Ocean will display a box offering to
 
 image:blueocean/intro/new-pipeline-box.png[Blue Ocean Create Pipeline Message Box, role=center]
 
+[[navigation-bar]]
+== Navigation bar
 
-=== Switching to the "Classic" UI
+Blue Ocean uses a common Navigation bar at the top of most Blue Ocean views.
+It includes five buttons:
+
+* *Jenkins* - Navigate to the Dashboard (reload if already viewing it)
+* *Pipelines* - Navigate to the Dashboard (do nothing if already viewing it)
+* *Administration* - <<../managing#, Manage>> this Jenkins instance (using the Classic UI)
+* *Switch to "Classic" UI* (arrow and square icon) - <<#switch-to-classic, Switch to the "Classic">>  Jenkins UI
```

Comment:
@i386 - it is shown below in the linked section.  `blueocean/intro/switch-classic.png` .